resources/js/features/home/components/+root/RecentForumPosts/RecentForumPostItem.tsx

```diff
@@ -48,7 +48,7 @@ export const RecentForumPostItem: FC<RecentForumPostItemProps> = ({ post }) => {
         />
       </p>
 
-      <p className="line-clamp-1">{post.latestComment.body}</p>
+      <p className="line-clamp-1 lg:max-w-[580px] xl:max-w-[816px]">{post.latestComment.body}</p>
```

Comment:
I don't have a monitor that supports large resolutions, but is 816px still appropriate at 4K?

Reply:
Yep, this is driven by the width of the container and independent of DPI or resolution.